src/support/azext_support/custom.py

```diff
@@ -0,0 +1,151 @@
+
+    return client.update(support_ticket_name=ticket_name, severity=severity, contact_details=body)
+
+
+def list_support_tickets_communications(cmd, client, ticket_name=None, filters=None):
+    return client.list_by_subscription_ticket(support_ticket_name=ticket_name, filter=filters)
+
+
+def get_support_tickets_communications(cmd, client, ticket_name=None, communication_name=None):
+    return client.get_by_subscription_ticket(support_ticket_name=ticket_name, communication_name=communication_name)
+
+
+def create_support_tickets(cmd, client,
+                           ticket_name=None,
+                           problem_classification=None,
+                           title=None,
+                           description=None,
+                           severity=None,
+                           start_time=None,
+                           require_24_by_7_response=None,
+                           contact_first_name=None,
+                           contact_last_name=None,
+                           contact_method=None,
+                           contact_email=None,
+                           contact_additional_emails=None,
+                           contact_phone_number=None,
+                           contact_timezone=None,
+                           contact_country=None,
+                           contact_language=None,
+                           technical_resource=None,
+                           quota_change_version=None,
+                           quota_change_subtype=None,
+                           quota_change_regions=None,
+                           quota_change_payload=None,
+                           partner_tenant_id=None):
+    service_name = parse_support_area_path(problem_classification)["service_name"]
+    service = "/providers/Microsoft.Support/services/{0}".format(service_name)
+
+    contactBody = {}
+    contactBody["first_name"] = contact_first_name
+    contactBody["last_name"] = contact_last_name
+    contactBody["preferred_contact_method"] = contact_method
+    contactBody["primary_email_address"] = contact_email
+    contactBody["additional_email_addresses"] = contact_additional_emails
+    contactBody["phone_number"] = contact_phone_number
+    contactBody["preferred_time_zone"] = contact_timezone
+    contactBody["country"] = contact_country
+    contactBody["preferred_support_language"] = contact_language
+
+    body = {}
+    body["description"] = description
+    body["problem_classification_id"] = problem_classification
+    body["severity"] = severity
+    body["contact_details"] = contactBody
+    body["title"] = title
+    body["service_id"] = service
+    body["require24_x7_response"] = require_24_by_7_response if require_24_by_7_response is not None else False
+    start_date_time = start_time if start_time is not None else datetime.now()
+    start_date_time = start_date_time.strftime("%Y-%m-%dT%H:%M:%SZ")
+    body["problem_start_time"] = start_date_time
```

Comment:
no SDK wrapper for `contactBody` and `body`?

Reply:
As discussed over chat, these models are generated from auto rest and api swagger based on arm recommendation.